src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -232,6 +239,9 @@ const AxisClipSliders: React.FC<AxisClipSlidersProps> = (props) => {
 
   const create3dAxisSlider = (axis: AxisName): React.ReactNode => {
```

Comment:
When is a `3dAxisSlider` used vs. a `2dAxisSlider`? Trying to understand why this was returning null.